src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -961,6 +963,23 @@ def _wait_for_schedule_changes_to_apply(self):
             if schedule_desc["MonitoringScheduleStatus"] != "Pending":
                 break
 
+    def validate_network_config(self, network_config_dict):
+        """Validates that EnableInterContainerTrafficEncryption is not set in
+        NetworkConfig object
+
+        """
+        if "EnableInterContainerTrafficEncryption" in network_config_dict:
+            message = (
+                "EnableInterContainerTrafficEncryption is not supported in Model Monitor. "
+                "Please ensure that encrypt_inter_container_traffic=None "
+                "when creating your NetworkConfig object. "
+                "Current encrypt_inter_container_traffic value: {}".format(
+                    self.network_config.encrypt_inter_container_traffic
+                )
+            )
+            print(message)
```

Comment:
use a logger instead of a print statement so that the user can control what log messages are printed to the console